src/index.js

```diff
@@ -76,20 +78,54 @@ function displayBoard() {
     'T': 'tail',
     'P': 'pellet'
   }
+
   game.gameBoard.getBoard().forEach(function (rowData) {
     var row = document.createElement('tr');
+    let rowIndex = 0
+    const rows = game.gameBoard.getRows()
+    const columns = game.gameBoard.getColumns()
     rowData.forEach(function (cellData) {
       var cell = document.createElement('td');
       cell.className = boardClassNameMap[cellData]
+      cell.id = determineCellBorder(rowIndex, columnIndex, rows, columns)
       row.appendChild(cell);
+      rowIndex += 1
     });
     tableBody.appendChild(row);
+    columnIndex += 1
   });
   table.appendChild(tableBody);
   document.querySelector('.game').appendChild(table)
   displayController();
 }
 
+const determineCellBorder = (rowIndex, columnIndex, rows, columns) => {
+  if (rowIndex === 0 && columnIndex === 0) {
```

Comment:
topLeftcell is just top and left... maybe there's a better way to clean this up (same sorta thing goes on in the CSS classs names)

Reply:
How do I determine that in my function? If its a corner cell, taking out the special edge cases would result in missing spots. 
<img width="765" alt="Screen Shot 2020-08-19 at 10 07 47 PM" src="https://user-images.githubusercontent.com/61174647/90708687-6bb93f00-e268-11ea-88cd-64148bfcd387.png">

Reply:
Hmm I got an idea I'll try out